app/services/notices/setup/determine-notice-type.service.js

```diff
@@ -0,0 +1,104 @@
+'use strict'
+
+/**
+ * Determine the notice type
+ * @module DetermineNoticeTypeService
+ */
+
+const { generateRandomInteger } = require('../../../lib/general.lib.js')
+
+/**
+ * Defines the configuration for supported notification types.
+ *
+ * This structure enables consistent handling and display of different notification types
+ * across the system. Each type includes legacy fields (`name` and `subType`) required for backwards compatibility.
+ *
+ * Legacy context:
+ * - `name` is used in the legacy UI to render the notification type on `/notifications/report`
+ * - `subType` is used when querying notifications in the legacy system
+ *
+ * @private
+ */
+const NOTIFICATION_TYPES = {
+  invitations: {
+    journey: 'invitations',
+    name: 'Returns: invitation',
+    prefix: 'RINV-',
+    redirectPath: 'returns-period',
+    subType: 'returnInvitation',
+    type: 'Returns invitation'
+  },
+  reminders: {
+    journey: 'reminders',
+    name: 'Returns: reminder',
+    prefix: 'RREM-',
+    redirectPath: 'returns-period',
+    subType: 'returnReminder',
+    type: 'Returns reminder'
+  },
+  'ad-hoc': {
+    journey: 'ad-hoc',
+    name: 'Returns: ad-hoc',
+    prefix: 'ADHC-',
+    redirectPath: 'licence',
+    subType: 'adHocReminder',
+    type: 'Ad hoc'
+  },
```

Comment:
Just checking. Is this still in here because this is a first step in refactoring 'ad-hoc' out?